src/schemas/22-0994/schema.js

```diff
@@ -0,0 +1,187 @@
+import _ from 'lodash/fp';
+import { countries } from '../../common/constants';
+import definitions from '../../common/definitions';
+import {
+  documentTypes526,
+  pciuCountries,
+  pciuStates
+} from '../../common/constants';
+
+const vetTecPrograms = [
+  "NAME",
+  "NAME",
+  "NAME",
+  "NAME",
+  "NAME"
+];
+
+const highTechnologyEmploymentType = [
+  "Computer Programming",
+  "Data Processing",
+  "Computer Software",
+  "Information Sciences",
+  "Media Application"
+];
+
+const salaries = [
+  "<$20,000",
+  "$20,001 - $35,000",
+  "$35,001 - $50,000",
+  "$50,001 - $75,000",
+  ">$75,000"
+];
```

Comment:
I think all of these enum values can be defined in the property since they won't be used anywhere else since they're only used in one place.  Also, I think we should change these to something that is more "enum-like" (so something like `computerProgramming`, `dataProcessing`, or `lessThanTwenty` and then vets-api will map them to the form and the display values will be defined in the `vets-website` ui schema.